### PR TITLE
added mapper and api resources

### PR DESCRIPTION
**Pull request by NicoleSauer**
- changed to much 🙃

### CODE REVIEW

File: src/classes/category.py [hunk 1]
```diff
@@ -2,41 +2,40 @@
...
+
+    def get_guideline_for_ten(self) -> str:
+        return self._guideline_for_ten
+
+    def set_guideline_for_ten(self, guideline_for_ten) -> None:
+        self._guideline_for_ten = guideline_for_ten
+
+    def __str__(self) -> str:
+        return (
+            f'Vacancy ID: {self._id}\n'  # Shouldn´t this be Category ID?
```
Comment (ScottPalme) [inline src/classes/category.py:37]: Correct this should be Category ID. Probably Copy paste mistake.

File: .gitignore [hunk 2]
```diff
@@ -168,3 +168,4 @@ data
 # DS_Store files
 .DS_Store
 src/apis/uploads/
+src/DS_Store
```
Comment (ScottPalme) [inline .gitignore:171]: The DS-Store files should be excluded, but I can still see one DS_Store file inside the services directory and only, when i view the Pull-Request inside VS-Code. Is this just a Bug or does it still track these files?

File: src/classes/category.py (hunk 1 above)
Comment (NicoleSauer) [reply src/classes/category.py:37]: Changed it to Category.

File: .gitignore (hunk 2 above)
Comment (NicoleSauer) [reply .gitignore]: These files were already tracked by Git, so this line had no effect. I now removed them and adjusted the gitignore accordingly. Should be fine now 👍🏻

**Review by ScottPalme — APPROVED** [review]

**Pull request merged**